Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -96,16 +95,28 @@ function Main() {
 		LogMsg "$?: Installed required packages, $req_pkg"
 
 		# Start kernel compilation
-		LogMsg "Clone and compile new kernel from $hb_url to /usr/src/linux"
-		git clone $hb_url /usr/src/linux
-		LogMsg "$?: Cloned the kernel source repo in /usr/src/linux"
+		LogMsg "Cloning a new kernel from $hb_url to $linux_path"
+		git clone $hb_url $linux_path
+		LogMsg "$?: Cloned the kernel source repo in $linux_path"
 
-		cd /usr/src/linux/
+		cd $linux_path
 
 		git checkout $hb_branch
 		LogMsg "$?: Changed to $hb_branch"
 
-		cp /boot/config*-azure /usr/src/linux/.config
+		if [[ "$DISTRO" =~ "redhat" || "$DISTRO" =~ "centos" ]];then
+			cp /boot/config* $linux_path/.config
```

Comment:
There is a possibility of more than 1 /boot/config* file present if we run this testcase with "-CustomKernel" parameter. 
You can add "config-$(uname -r)" to get single file.

Reply:
- CustomKernel won't work this test. This requires specific version and compile one inside. Using CustomKernel does not make sense,